2017-2018/gameEngine/messages/__init__.py

```diff
@@ -4,8 +4,8 @@
 
 class MsgType(Enum):
     LIGHT_STATE = 0
-    FULL_STATE = 1
-    PACMAN_LOCATION = 2
+    PACMAN_LOCATION = 1
```

Comment:
😞

Reply:
I made this change, so the enums would match up between the server and the bot code. We're using the framework in a weird way, because it was not designed to have modules connect to 2 separate servers. I will fix this in a future version of the framework